CMakeLists.txt

```diff
@@ -127,7 +127,9 @@ add_definitions(
 # DLL goop on Windows.
 #add_definitions(-DANGLE_EXPORT="")
 
-include_directories(include src src/common/third_party/base)
+find_package(X11 REQUIRED)
```

Comment:
Hmm, how does this even work for non-unix-like platforms?

Reply:
It's not marked as `REQUIRED` so it will just not find it and go on I think

Reply:
Oh, right. Marked as not `REQUIRED` now.